### PR TITLE
added small additions, corrections of formatting

### PR DESCRIPTION
**Pull request by Neodelf**
Hello, Bozhidar. This prospect has added some changes to direct and change the formatting of some of the blocks.
If you think that any changes unnecessary, please comment on them, and I will make changes.
Thank you for creating such a guide!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1946,10 +1966,10 @@ no parameters.
 
   ```Ruby
   # bad - identifier using non-ascii characters
-  заплата = 1_000
+  зарплата = 1_000
```
Comment (bbatsov) [inline README.md]: I've used Bulgarian, not Russian. :-)

File: README.md [hunk 2]
```diff
@@ -1946,10 +1966,10 @@ no parameters.
...
 
   # bad - identifier is a Bulgarian word, written with Latin letters (instead of Cyrillic)
-  zaplata = 1_000
+  zarplata = 1_000
```
Comment (bbatsov) [inline README.md]: As noted above... :-)

File: README.md [hunk 3]
```diff
@@ -2460,7 +2482,7 @@ no parameters.
 <sup>[[link](#attr)]</sup>
 
   ```Ruby
-  # bad - creates a single attribute accessor (deprecated in 1.9)
+  # bad - creates a single attribute accessor (deprecated in ruby version 1.9)
```
Comment (bbatsov) [inline README.md]: ruby version -> Ruby

File: README.md [hunk 4]
```diff
@@ -1148,19 +1148,39 @@ condition](#safe-assignment-in-condition).
...
+  x = Math.sin y
+  # good
   x = Math.sin(y)
+
+  # bad
+  array.delete e
+  # good
   array.delete(e)
 
+  # bad
```
Comment (bbatsov) [inline README.md]: I get the point of the final example, but this will generate a warning. That's one of the reasons `rspec` moved away from this "syntax". Using `eq` won't produce a warning.

**Comment by bbatsov** [comment]
A note about the commit message - please reword it to "Improve some examples and apply some formatting corrections".

File: README.md (hunk 4 above)
Comment (Neodelf) [inline README.md]: Well, then I suggest to do
```
# bad
expect(bowling.score).to eq 0
# good
expect(bowling.score).to eq(0)
```
What do you think about it?

File: README.md (hunk 1 above)
Comment (Neodelf) [inline README.md]: Of course, Bulgarian! I`m so sorry about this fix :-)

File: README.md (hunk 3 above)
Comment (Neodelf) [inline README.md]: Ok

**Comment by Neodelf** [comment]
Yes, I will reword commit message, but please answer [this question](https://github.com/bbatsov/ruby-style-guide/pull/551/files#r57509561)

File: README.md (hunk 4 above)
Comment (bbatsov) [inline README.md]: Fine by me.

Comment (bbatsov) [inline README.md]: To replace `bowling.score.should.==(0)` you can use simpler examples such as `x + y` & `x.+(y)`.

Comment (Neodelf) [inline README.md]: Because this paragraph about the use of parentheses, I suggest to add
```
# bad
x +(y)
# good
x + y

File: README.md [hunk 5]
```diff
@@ -1148,20 +1148,43 @@ condition](#safe-assignment-in-condition).
...
   array.delete(e)
 
-  bowling.score.should == 0
+  # bad
+  expect(bowling.score).to eq 0
+  # good
+  expect(bowling.score).to eq(0)
+
+  # bad
+  x+(y)
```
Comment (bbatsov) [inline README.md]: Not sure what this illustrates, but I don't think it belongs here. Seems like it illustrates redundant parens.

File: README.md (hunk 4 above)
Comment (bbatsov) [inline README.md]: I don't think this will fit well with the other examples. Guess we can just skip this.

File: README.md (hunk 5 above)
Comment (Neodelf) [inline README.md]: Yes, it shows extra parentheses so it's bad. This row better illustrated that `.` is a method. You think I should remove `x+(y)` and `x+y`?

Comment (bbatsov) [inline README.md]: Yeah, remove this.

**Comment by Neodelf** [comment]
@bbatsov  I think all the issues have been resolved

**Comment by bbatsov** [comment]
:+1:
